draft-ietf-httpbis-priority.md

```diff
@@ -177,10 +177,11 @@ requiring less interpretation and allowing less variation.
 RFC 7540 does not define a method that can be used by a server to provide a
 priority signal for intermediaries.
 
-RFC 7540 priority is expressed relative to other requests on the same connection.
-Many requests are generated without knowledge of how other requests might share a
-connection, which makes this difficult to use reliably, especially in protocols
-that do not have strong ordering guarantees, like HTTP/3 {{HTTP3}}.
+RFC 7540 priority is expressed relative to other requests sharing the same
+connection at the same time. It is difficult to incorporate such design into
+applications that generate requests without knowledge of how other requests
+might share a connection, or into protocols that do not have strong ordering
+guarantees, like HTTP/3 {{HTTP3}}.
```

Comment:
maybe "ordering guarantees across streams" or "ordering guarantees between requests" would be even clearer

Reply:
Good point! Went with "across streams," as H2 does not have ordering guarantee between _requests_.